src/istio/authn/context.proto

```diff
@@ -33,17 +35,13 @@ message JwtPayload {
   // id. Example 123456789012.my-svc.com
   string presenter = 3;
 
-  // Only raw JWT string claims are kept.
-  map<string, string> claims = 5;
+  // JWT claims stored as protobuf.Struct
```

Comment:
Keep `comments that only string and stringlist are kept. String claims are kept string string list of one item`,

Reply:
Done.